engine/execution/ingestion/engine.go

```diff
@@ -1372,9 +1374,27 @@ func (e *Engine) fetchCollection(blockID flow.Identifier, height uint64, guarant
 		)
 		return fmt.Errorf("could not find guarantors: %w", err)
 	}
+
+	filters := []flow.IdentityFilter{
+		filter.HasNodeID(guarantors...),
+	}
+
+	// This is included to temporarily work around an issue observed on a small number of ENs.
+	// Using only the onflow LNs seems to aleviate the issue. This will be removed once a
+	// proper fix is in place.
+	if os.Getenv("ONFLOW_LN_ONLY_OVERRIDE") != "" {
+		var onlyOnflowRegex = regexp.MustCompile(`.*onflow.org:3569$`)
```

Comment:
```suggestion
		var onlyOnflowRegex = regexp.MustCompile(os.Getenv("ONFLOW_LN_ONLY_OVERRIDE"))
```

are we supposed to use from env var?

Reply:
no, I was thinking it would just be set to `ONFLOW_LN_ONLY_OVERRIDE=1` or something like that.